.github/workflows/copilot-setup-steps.yml

```
@@ -0,0 +1,29 @@
+
+jobs:
+  copilot-setup-steps:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v6
+      - uses: ./.github/actions/setup-bun
+        with: { install-node: "true" }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)